config/impl/src/main/java/org/jboss/windup/config/loader/WindupRuleLoaderImpl.java

```diff
@@ -64,49 +62,6 @@ public WindupRuleMetadata loadConfiguration(GraphContext context, Predicate<Wind
         }
 
         List<WindupRuleProvider> providers = WindupRuleProviderSorter.sort(allProviders);
-        for (WindupRuleProvider provider : providers)
```

Comment:
Thank you... although I wonder if we could put this into a utility class somewhere? test-utils perhaps?